applications/FluidDynamicsApplication/custom_processes/shock_capturing_entropy_viscosity_process.cpp

```diff
@@ -337,7 +339,8 @@ std::tuple<ShockCapturingEntropyViscosityProcess::TotalDerivativeUtil, ShockCapt
 ShockCapturingEntropyViscosityProcess::BuildTotalDerivativeUtils(
     const Element& rElement,
     const double DeltaTime,
-    const double HeatCapacityRatio)
+    const double gamma,
+    const double c_v)
```

Comment:
```suggestion
    const double Gamma,
    const double HeatCapacityConstVol)
```
Note the style guide changes (CamelCase for functions arguments) and the usage of descriptive argument names.

Reply:
You can also use `HeatCapacityRatio` for gamma as you did below.